recipes/mapbox-vector-tile/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mapbox-vector-tile" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 445e7473ee0f4d2916e6952f6f9402e081d55a9ce0d694a047d767aee8c1ac0c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+
+  run:
+    - future
+    - protobuf
+    - pyclipper
+    - python
+    - setuptools
+    - shapely
+
+test:
+  imports:
+    - mapbox_vector_tile
+    - mapbox_vector_tile.Mapbox
+    - tests
```

Comment:
This import shouldn't be provided by the package else would have conflicts with other packages.

The `exclude` argument to `find_packages` is probably missing upstream in `setup.py`. Can you patch it?
```python
find_packages(exclude=['tests'])
```

Reply:
Patch has been added. Thanks.